cmd/nvidia-container-runtime-hook/main.go

```diff
@@ -89,6 +89,16 @@ func doPrestart() {
 	rootfs := getRootfsPath(container)
 
 	args := []string{getCLIPath(cli)}
+
+	// Opt out of the fabricmanager and persistenced sockets if these are not
+	// explicitly enabled.
+	if !hook.Features.AllowFabricmanager.IsEnabled() {
```

Comment:
So, containers depending on this socket (e.g. NCCL library) will error out after this update by default? This will impact multi-node training Jobs etc.

Reply:
Changing the scope of this PR to only `nvidia-persistenced`.